google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -44,15 +48,22 @@
 import java.util.List;
 import java.util.Set;
 import java.util.Timer;
-import java.util.TimerTask;
 
 /**
  * Queries multiple states on a fixed interval for updates. It notifies listeners
  * when updates occur.
  */
 public class CloudDebugGlobalPoller {
+
   private static final int DELAY_MS = 5000;
   private static final Logger LOG = Logger.getInstance(CloudDebugGlobalPoller.class);
+
+  /**
+   * Display group used to display notifications in the IDE
+   */
+  public static final String CLOUD_DEBUGGER_ERROR_NOTIFICATIONS_DISPLAY_GROUP =
+      "Cloud Debugger Error Notifications";
```

Comment:
That's right, it is used only internally by IntelliJ to differentiate between different notification groups allowing to set different behavior for them.